packages/zig.rb

```diff
@@ -3,22 +3,29 @@
 class Zig < Package
   description 'Programming language designed for robustness, optimality, and clarity'
   homepage 'https://ziglang.org/'
-  version '0.7.1'
+  version '0.8.1'
   license 'MIT'
-  compatibility 'x86_64 aarch64'
-  source_url 'https://ziglang.org/download/0.7.1/zig-0.7.1.tar.xz'
```

Comment:
Building from source as the current version does?

Reply:
No need.  The binaries provided work fine.  I always appreciate it when projects provide working binaries for all architectures.

Reply:
And...compatibility is all now.